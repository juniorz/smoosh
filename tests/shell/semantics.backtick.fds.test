set -e
subshfds=$($TEST_UTIL/fds 0 20)
$TEST_UTIL/fds 0 20
echo $subshfds
