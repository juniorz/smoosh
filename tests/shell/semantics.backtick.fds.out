0 open
1 open
2 open
3 closed
4 closed
5 closed
6 closed
7 closed
8 closed
9 closed
10 closed
11 closed
12 closed
13 closed
14 closed
15 closed
16 closed
17 closed
18 closed
19 closed
20 closed
0 open 1 open 2 open 3 closed 4 closed 5 closed 6 closed 7 closed 8 closed 9 closed 10 closed 11 closed 12 closed 13 closed 14 closed 15 closed 16 closed 17 closed 18 closed 19 closed 20 closed
